### PR TITLE
Remove unwanted \

### PR DESCRIPTION
**Pull request by peabee**
causes new grep WARNINGS

### CODE REVIEW

**Review by 01micko — DISMISSED** [review]

File: woof-code/0setup [hunk 1]
```diff
@@ -68,7 +68,7 @@ if [ "$1" = "" ] ; then
 fi
 #======================================================================
 
-if wget --help | grep -q '\-\-show\-progress' ; then
+if wget --help | grep -q '--show-progress' ; then
```
Comment (dimkr) [inline woof-code/0setup:71]:
```
~$ wget --help | grep -q '--show-progress'
grep: unrecognized option '--show-progress'
Usage: grep [OPTION]... PATTERNS [FILE]...
Try 'grep --help' for more information.
```

Comment (dimkr) [reply woof-code/0setup:71]: You'll need to be very careful.

Comment (dimkr) [reply woof-code/0setup:71]: You need `--` to stop option parsing, i.e. `wget --help | grep -q -- '--show-progress'`

**Comment by 01micko** [comment]
There's probably many more. It _was_ standard practice to escape `-` especially if the 'grep' arg was surrounded in `".."`

File: woof-code/0setup (hunk 1 above)
Comment (01micko) [reply woof-code/0setup:71]: Oh, so there _is_ a case for escape.

Comment (dimkr) [reply woof-code/0setup:71]: Yes, but you don't need that escaping if you make grep understand this is not an option the right way 😄

Comment (01micko) [reply woof-code/0setup:71]: Also leaving off the '--' works

`wget --help | grep -q 'show-progress'`

**Comment by dimkr** [comment]
> There's probably many more. It _was_ standard practice to escape `-` especially if the 'grep' arg was surrounded in `".."`

See #3397

**Comment by peabee** [comment]
Replaced by #3397

**Pull request closed without merging**